cmd/subnetcmd/import_from_api.go

```diff
@@ -0,0 +1,282 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package subnetcmd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/ava-labs/avalanche-cli/pkg/constants"
+	"github.com/ava-labs/avalanche-cli/pkg/models"
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+	"github.com/ava-labs/avalanche-cli/pkg/vm"
+	"github.com/ava-labs/avalanchego/api/info"
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/rpc"
+	"github.com/ava-labs/avalanchego/vms/platformvm"
+	"github.com/ava-labs/coreth/core"
+	"github.com/spf13/cobra"
+)
+
+var (
+	genesisFilePath string
+	subnetIDstr     string
+	nodeURL         string
+)
+
+// avalanche subnet import
+func newImportFromNetworkCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "import-running [subnetPath]",
```

Comment:
Hmm, maybe we could. However, due to the independence of the flags for each, this can quickly become very confusing, requiring multiple checks for mutual exclusion (i.e. the flags required for one can't be set for the other and vice-versa).

Reply:
Your suggestion of making two separate sub-commands seems best.